source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -354,12 +354,14 @@ locally-managed master key.
 C. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-`JSON Schema <http://json-schema.org/>`_ is a vocabulary that allows you
-to annotate and validate JSON documents. MongoDB drivers use an extended
-version of the JSON Schema standard to configure automatic client-side
-encryption and decryption of specific fields of the documents in a
-collection. The extended MongoDB CSFLE standard requires the following
-information:
+In this section, MedcoMD engineers configure the fields that the client
+automatically encrypts and decrypts using JSON Schema.
+`JSON Schema <http://json-schema.org/>`_, a vocabulary
```

Comment:
Grammar is off here. Either combine the two sentences or add an **is**

JSON Schema **is** a vocabulary...

Reply:
Oops, will fix.